MinMaxGame.java

```diff
@@ -1,99 +1,99 @@
 import java.util.Scanner;
 
 class MinMaxGame extends Game {
+	// Implements the negamax tree search algorithm.
```

Comment:
Maybe write a bit more in this docstring explaning what this is doing

Reply:
It would be complicated.